projects/ui/src/shim.cds-core.scss

```diff
@@ -752,12 +764,6 @@
     }
   }
 
-  .btn:not([disabled]):hover,
```

Comment:
@colinreedmiller One thing to consider here is that users may expect all buttons across their application to apply these values, so I wanted to double check that we're comfortable removing these values from the shim.

Reply:
Without usage data the question may be more about whether or not we can support the requests for assistance when teams encounter unexpected results. It may be wiser to retain these values until we have a migration plan that addresses this.